src/server/server_family.cc

```diff
@@ -458,10 +458,11 @@ ServerFamily::ServerFamily(Service* service) : service_(*service) {
 ServerFamily::~ServerFamily() {
 }
 
-void SetMaxClients(std::vector<facade::Listener*>& listeners, uint32_t maxclients) {
+template <typename F>
+void SetMaxClients(std::vector<facade::Listener*>& listeners, uint32_t maxclients, F f) {
```

Comment:
std::function would have done the job 🙃

Reply:
Nah, PTSD kicks in when I can avoid type erasure. Not that it matters in this context, but oh well :shrug: